components/pagination/pagination.tsx

```diff
@@ -29,6 +29,10 @@ interface Props {
   size?: NormalSizes
   defaultPageSize?: number
   pageSizeOptions?: string[]
+  labelPageSizeChangerBefore?: ReactNode | string
```

Comment:
is the name too long？ how about labelPageSizeBefore labelJumptoBefore

Reply:
done